### PR TITLE
People json

### PR DESCRIPTION
**Pull request by kiminlee2000225**

### CODE REVIEW

File: src/components/PeopleData.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
+const people = [
```
Comment (jackblanc) [inline src/components/PeopleData.tsx]: We want to have this as a typed object:
```
interface Person {
  readonly name: string;
  readonly position: string;
  ....
}
```

Then have: 
`const people: People[] = [...`

File: src/components/People.tsx [hunk 2]
```diff
@@ -61,62 +62,7 @@ const People: React.FC = () => {
...
-                email="tonina.j@northeastern.edu"
-              />
-              <Member
-                name="Justin Konecny"
-                position="Technical Lead"
-                image="https://c4cneu-public.s3.us-east-2.amazonaws.com/Board/2020/Justin-K.jpg"
-                linkedin="https://www.linkedin.com/in/justin-konecny/"
-                email="konecny.j@northeastern.edu"
-              />
+              {PeopleData.map(person => <Member name={person.name} position={person.position} image={person.image} linkedin={person.linkedin} email={person.email} />)}
```
Comment (jackblanc) [inline src/components/People.tsx]: The `Member` component should take as props a `Person`.
You can do `<Member person={person} />` (might have to give them different names)

**Review by jackblanc — CHANGES_REQUESTED** [review]
Let me know if you have any questions on the Typescript stuff 👍

**Comment by rymaju** [comment]
See #40

**Pull request closed without merging**